scripts/cpp_generator.py

```diff
@@ -57,6 +57,15 @@
     "XrBaseOutStructure",
     # Array of XrColor4f not getting initialized right
     "XrPassthroughColorMapMonoToRgbaFB",
+    # No "invalid" atom value
+    'XrAsyncRequestIdFB',
+    'XrEventDataSpatialAnchorCreateCompleteFB',
+    'XrEventDataSpaceSetStatusCompleteFB',
+    'XrEventDataSpaceQueryCompleteFB',
+    'XrEventDataSpaceEraseCompleteFB',
+    'XrEventDataSpaceQueryResultsAvailableFB',
+    'XrEventDataSpaceSaveCompleteFB',
+    'XrSpaceSaveInfoFB',
```

Comment:
Not sure if I understand correctly...
Since we have implemented non-nullable atoms, is it safe to remove these atoms from skip list?

I tried to generate headers with this commented out, and all tests are compiled successfully and passing.

Reply:
I think it is... I didn't get this far yesterday, ran out of time.

Reply:
Yeah, they look good, I have re-enabled them.